common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -115,4 +115,14 @@ trait ABTestSwitches {
     sellByDate = new LocalDate(2020, 9, 4),
     exposeClientSide = true
   )
+
+  Switch(
+    ABTests,
+    "ab-remote-render-epic",
+    "A/B test local vs remote render of default epic, to validate Slot Machine approach and work to date",
+    owners = Seq(Owner.withGithub("tjmw"), Owner.withGithub("nicl")),
+    safeState = On,
+    sellByDate = new LocalDate(2020, 2, 4),
```

Comment:
Reminder that we need to update this date 📆

Reply:
thanks yeah! Maybe til the end of the month?